analyses/mutational-signatures/util/mut_sig_functions.R

```diff
@@ -110,38 +115,48 @@ bubble_matrix_plot <- function(sig_num_df, label = "none") {
   #   sig_num_df: a data.frame with number of mutations per Mb that belong to
   #               each sample x signature combo
   #   label: a character string for the title of the plot to be passed to ggplot2::ggtitle
-  #
+  #   color_palette: a set of colors to use for a color palette. 
+  #   color_breaks: a set of numeric breaks to be used for color palette, needs to be 
+  #                 the same length as the color palette provided. 
+  #    
   # Returns:
   #   A bubble matrix plot with the number of mutations per Mb and proportion of 
   #   tumors with a non-zero weight for all samples summarized by histology.
-  #
+  
   # Summarize the mut_per_mb column by histology
   grouped_sig_num <- sig_num_df %>%
     dplyr::group_by(short_histology, signature) %>%
     dplyr::summarize(
       # Calculate the proportion of tumors with a non-zero weight for each signature
       prop_tumors = sum(num_mutations > 0) / length(num_mutations),
       # Calculate the median number of mutations per Md
-      med_num = median(mut_per_mb)
+      med_num = median(mut_per_mb, na.rm = TRUE)
     )
   
   # Turn 0's into NAs so they are not actually plotted
   grouped_sig_num$prop_tumors[which(grouped_sig_num$prop_tumors == 0)] <-  NA
-
+  
+  # If color palette is not specified use color brewer's YlGnBu
+  if (is.na(color_palette[1])) {
```

Comment:
```suggestion
  if (any(is.na(color_palette))) {
```
might accomplish what you want and be more robust?